src/main.rs

```diff
@@ -114,12 +114,20 @@ fn main() {
 
             info!("Sent command: Run");
             replies.push(comm);
-            enclave_process_handle_all_replies::<EnclaveRunInfo>(&mut replies, 0, false, vec![0])
+            let run_info = enclave_process_handle_all_replies::<EnclaveRunInfo>(&mut replies, 0, false, vec![0])
                 .map_err(|e| {
                     e.add_subaction("Failed to handle all enclave process replies".to_string())
                         .set_action(RUN_ENCLAVE_STR.to_string())
                 })
                 .ok_or_exit_with_errno(None);
+            if run_args.attach_console.unwrap_or(false) {
+                console_enclaves(run_info[0].enclave_cid, None)
```

Comment:
There is an assumption here is that there's 1 enclave process which contains a valid CID, which I think will always be true, but I'm not sure that if there's more than 1 enclave process, the one found at index 0 will be the one that was just started. @aatrand what do you think?

Reply:
The array elements in `run_info` map 1:1 to the elements in `replies`. `replies` is initialized in this function with exactly 1 element.